### DISCUSSION TITLE
School

### DISCUSSION THREAD

**Discussion by Shadvaladi** [community/community#17719] [GitHub Education]
I'm unable to find my correct school which is in Renukoot.
My school name is Nirmala Convent High School (https://www.ncsrenukoot.org/) in Renukoot and I didn't find so I selected other school of same name.
Please try to help me.
Thank you

**Comment by mlarasusan** [comment]
Hello [Shadvaladi](https://github.com/Shadvaladi). If your institution doesn’t qualify you for the Pack, ask an instructor to apply for your school to become a [GitHub Campus partner](https://education.github.com/schools). Thanks :]

**Comment by Paapaapa** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
